packages/playground/src/utils/filter_farms.ts

```diff
@@ -0,0 +1,49 @@
+import type { InputFilterType } from "../types";
+import { isAlphanumeric, isInt, isNumeric, min, validateResourceMaxNumber } from "./validators";
+
+export type FilterFarmInputs = {
+  farmId: InputFilterType;
+  name: InputFilterType;
+  pricingPolicyId: InputFilterType;
+  totalIps: InputFilterType;
+};
+export const inputsInitializer: FilterFarmInputs = {
+  farmId: {
+    label: "Farm IDs",
+    placeholder: "e.g. 1,2,3.",
+    rules: [
+      [isNumeric("Should be a number"), min("The ID should be larger then zero.", 1), isInt("should be an integer")],
+    ],
+    type: "text",
+  },
+  name: {
```

Comment:
The farm name filter is not working correctly as it only returns the farm if the exact name is entered. and if a part of the name was entered it return all farms even if the entered string doesn't match any of the returned farms.

[Screencast from 21-11-23 15:31:56.webm](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/101194226/92ce0293-3e5c-4970-bb6a-9f54d011cff0)